### PR TITLE
ruby package crawler for avoid_setns mode

### PR DESCRIPTION
**Pull request by sahilsuneja1**
using it requires --pluginmode to be fixed 

Signed-off-by: Sahil Suneja <sahilsuneja@gmail.com>

### CODE REVIEW

File: crawler/plugins/ruby_package_container_crawler.py [hunk 1]
```diff
@@ -15,40 +16,70 @@ class RubyPackageCrawler(IContainerCrawler):
...
+        for pkg in packages:
+            name_parts = re.match(r'(.*)-([\d\.]*)(\.gemspec)',pkg)
+            if name_parts is not None:
+                pkg_name = name_parts.group(1)
+                pkg_version = name_parts.group(2)
+                yield (
+                    pkg_name,
+                    {"pkgname": pkg_name, "pkgversion": pkg_version},
+                    'ruby-package')
+
```
Comment (ricarkol) [inline crawler/plugins/ruby_package_container_crawler.py]: remove empty lines

File: crawler/plugins/ruby_package_container_crawler.py [hunk 2]
```diff
@@ -15,40 +16,70 @@ class RubyPackageCrawler(IContainerCrawler):
...
+   
+    def _crawl_files(self,path,extension):
+        output=[]
+        if os.path.isdir(path):
+            for (root_dirpath, dirs, files) in os.walk(path):
+                output += [f for f in files if f.endswith(extension)]
+        return output
 
+        
+
```
Comment (ricarkol) [inline crawler/plugins/ruby_package_container_crawler.py]: too many empty lines. Run an autopep8 on this file.

File: crawler/plugins/ruby_package_container_crawler.py [hunk 3]
```diff
@@ -15,40 +16,70 @@ class RubyPackageCrawler(IContainerCrawler):
...
+                        yield (
+                            pkg_name,
+                            {"pkgname": pkg_name, "pkgversion": pkg_version},
+                            'ruby-package')
+
+   
+    def _crawl_files(self,path,extension):
+        output=[]
+        if os.path.isdir(path):
+            for (root_dirpath, dirs, files) in os.walk(path):
```
Comment (ricarkol) [inline crawler/plugins/ruby_package_container_crawler.py]: Are you sure you don't want to skip some directories here? thinking about performanc.e

Comment (ricarkol) [inline crawler/plugins/ruby_package_container_crawler.py]: Just realized that this is done just for the candidate paths. All good here.

**Comment by sahilsuneja1** [comment]
Autopep8'ed

**Comment by ricarkol** [comment]
lgtm

**Pull request merged**